### PR TITLE
[add] bert md file for report

### LINKED ISSUE #157: Add missing BERT report file
Report the best config from the latest leaderboard.

23:34:44,bert_hparam_trial_5,Andre_Hrs,b46b02f,trained_models/20260513_233250_bert_hparam_trial_5/bert_hparam_trial_5_model.pt,4,f1_macro,0.7062497138977051,0.7459935060391824,0.7875000238418579,0.7062497138977051,0.7959456443786621,0.6810367703437805,0.7513938546180725,5,2,9.04567407760267e-06,1.0,cross_entropy,False,BERT,BertClassifier,AdamW,StepLR,CrossEntropyLoss,True,110.2576904296875,0.86743920519948,0.7208333611488342,0.6205100417137146,0.7348974943161011,0.592323362827301,0.6799129843711853,0.8208333333333333,,True,0.8208333333333333,0.1583333333333333,0.0208333333333333,,,False,"{""class_5"": {""f1"": 0.0, ""target"": 0.7, ""met"": false}, ""class_6"": {""f1"": 0.0, ""target"": 0.7, ""met"": false}, ""class_11"": {""f1"": 0.0, ""target"": 0.7, ""met"": false}, ""class_17"": {""f1"": 0.0, ""target"": 0.7, ""met"": false}}","{""model_name"": ""bert-base-uncased"", ""pooling"": ""mean"", ""fine_tune"": true, ""batch_size"": 8, ""label_col"": ""potential_damage"", ""text_col"": ""description"", ""learning_rate"": 9.04567407760267e-06, ""dropout"": 0.15518824231537148, ""max_length"": 160, ""use_class_weights"": false, ""weight_decay"": 0.01}",False

talk a bit about SafetyBERT, but still consider the above config to be best recorded with 0.70 f1 score.

### PR DESCRIPTION
**Pull request by aashutosh-chapagain**
Closes #157

### CODE REVIEW

**Review by Mars-Martiny — APPROVED** [review]
Thanks for adding your report, this is good for transparency of our findings.
